include/ap_mmn.h

```diff
@@ -562,14 +562,18 @@
  * 20120211.106 (2.4.49-dev) Add ap_create_request().
  * 20120211.107 (2.4.49-dev) Add ap_parse_request_line() and
  *                           ap_check_request_header()
+ * 20120211.108 (2.4.49-dev) Add ap_normalize_path(),
+ *                           pre_translate_name hook and
+ *                           Add map_encoded_one and map_encoded_all bits to
+ *                           proxy_server_conf.
```

Comment:
Those were 3 separate MMN minor bumps in trunk, but I suppose that it's OK to merge them if they end up in a single 2.4.x backport..